.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+    "recommendations": [
+        "GuardRex.status-bar-tasks",
```

Comment:
I'm guessing there's other VS Code plugins that may be useful as recommendations, but I think these two cover the bare minimum of common ones for a TS project

Reply:
There aren't any specific extensions that I have been using in the development of this project, but I will add any that I come across.